CMakeLists.txt

```diff
@@ -270,12 +270,12 @@ if(GTEST_FOUND)
     GNAT
     QDEIM
     S_OPT
-    SVD
```

Comment:
Why is `SVD` removed?

Reply:
As mentioned above. `test_SVD` only tests the base SVD class, and only the time-interval related functionalities. The entire test routines become unnecessary as we remove the time interval.